package-lock.json

```
@@ -26,6 +26,7 @@
             },
             "devDependencies": {
                 "@microsoft/eslint-config-azuretools": "^0.1.0",
+                "@microsoft/vscode-azext-dev": "^0.1.2",
```

May as well do this (and `npm install`) to save a step later.
```suggestion
                "@microsoft/vscode-azext-dev": "^0.1.3",
```

---

Oh doh.  I got my trigger finger too happy.